### ISSUE TITLE
buf breakage issue

### ISSUE THREAD

**Issue by marbar3778** [bufbuild/buf#33]
hello again 😄 

I am running into an odd issue when a contributor pushes a pull request from their fork then the breakage checking can't find the master branch. you can see an example here: https://github.com/tendermint/tendermint/pull/4461 & https://circleci.com/gh/tendermint/tendermint/96231?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

should I specify the github repo as well, not sure if this is documented.

**Comment by Shivani912** [comment]
Thanks @marbar3778. I thought it was just me, whew :sweat_smile:

**Comment by bufdev** [comment]
You need to check against the remote, or manually pull the branch, before testing in CircleCI.

See https://github.com/bufbuild/buf-example:

```
The make targets make https and make ssh compare against the head of the remote git master branch, due to Travis and CircleCI not cloning any branch except for that under test. We use make https in Travis and make ssh in CircleCI.
```

Also see https://github.com/bufbuild/buf/issues/21

**Comment by marbar3778** [comment]
ah perfect thank you

**Comment by marbar3778** [comment]
hey you commented here: https://github.com/tendermint/tendermint/pull/4369#discussion_r385834579

I am a bit lost by what you mean by include thrid_party/proto in root, it is included in root.

**Comment by bufdev** [comment]
Tldr:

```yaml
build:
  roots:
    - proto
    - third_party/proto
```

Then, move your `.proto` source files to `proto/`, and remove `third_party/proto` from your import paths.

This is how https://github.com/bufbuild/buf/tree/master/proto is organized (internally, we have a `third_party/proto` directory as well).

```
proto
└── bufbuild
    ├── buf
    │   ├── file
    │   │   └── v1beta1
    │   │       └── file.proto
    │   └── image
    │       └── v1beta1
    │           └── image.proto
third_party
└── proto
    └── validate
        └── validate.proto
```
